demo/mobile/src/app/app.js

```diff
@@ -22,6 +25,112 @@ angular.module('wfm-mobile', [
   }),
   require('@raincatcher/vehicle-inspection')
 ]);
+
+var auth = {};
+
+// initialise the keycloak js adapter if keycloak configuration has been specified
+if (keycloakConfig) {
+  // require keycloak js client adapter
+  var Keycloak = require('keycloak-js');
+
+  // keycloak init config
+  var initConfig = {onLoad: 'login-required'};
+
+  // the keycloak json config
+  var keycloakJS = Keycloak(keycloakConfig);
+
+  angular.element(document).ready(function() {
```

Comment:
If you have bug reported (and I believe we have) we should link that as this may be actually resolved at some point. If not and that's not something that keycloak team will resolve let's wrap that in somehow reusable method so people will know what kinda problems this statement is going to resolve.